src/nfa/nfa.cc

```diff
@@ -217,13 +217,8 @@ namespace {
  *    SCC if useful, we declare all nodes in the SCC useful and moreover we propagate usefulness also the states
  *    in @p tarjan_stack as it contains states that can reach this closed SCC.
  *
- * @return BoolVector
- *
- * If @p stop_at_first_useful_state is true, then the algo stops at the first found useful state. This is used in an
- *  emptiness test.
  */
-BoolVector Nfa::get_useful_states(const bool stop_at_first_useful_state) const {
-    BoolVector useful(this->num_of_states(),false);
+void Nfa::tarjan_scc_discover(const TarjanDiscoverCallback& callback) const {
```

Comment:
Should the argument `callback` be mentioned/explained in the comment?

Reply:
Some text should be in the header file